Allow customization of the card element

OrangeJuiced opened the pull request:
https://github.com/199ocero/radio-deck/pull/7 added support for the customization of the options and descriptions. This PR extends that work by adding support for customization of the parent card element.

An example use case for this additional customization of the cards is to customize the rounding, ring width, etc.

OrangeJuiced committed:
Add extra card attributes to RadioDeck component

A new trait named 'HasExtraCardAttributes' has been added for the RadioDeck component. This trait allows for the addition of extra attributes to a card element using an array or a closure. Changes have also been made in the radio-deck.blade.php to utilize these extra card attributes.

OrangeJuiced committed:
Rename HasExtraCardAttributes to HasExtraCardsAttributes

The trait and associated methods 'HasExtraCardAttributes' have been renamed to 'HasExtraCardsAttributes' to better reflect their purpose. This naming change is propagated throughout the codebase, including the 'RadioDeck' component and the 'radio-deck.blade.php' view file.

199ocero commented:
Hey @OrangeJuiced  nice work and I'll gladly accept this PR but before I merge it could you also add in the read me how this pr will be use so that new users will know this feature is existing. Thank you!

OrangeJuiced committed:
Add extra card class attribute in README.md

The README.md has been updated to include the use of the 'extraCardsAttributes' property in the instructions for users. This property allows users to add extra class attributes to the card HTML element in the customizable radio deck implementation using TailwindCSS utilities.

OrangeJuiced commented:
Done! @199ocero :)
